style.css

```diff
@@ -109,12 +123,252 @@ h2 {
   color: solid #000;
   display: flex;
   list-style-type: none;
-  padding: 0;
+  padding: 0.05;
   justify-content: left;
   align-items: center;
   gap: 1.25rem;
 }
 
 .social-media a :hover {
-  fill: #3909c8;
+  color: #3909c8;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  grid-gap: 1.5rem;
+}
+
+.project-card {
+  padding: 0.75rem;
+  margin: 0.03 auto;
+  width: 100%;
+  max-width: 20.625rem;
+  border-radius: 0.05rem;
+}
+
+.project-title {
+  font-family: poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  letter-spacing: 0.03em;
+}
+
+.programming-languages {
+  color: solid #000;
+  display: flex;
+  list-style-type: none;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+}
+
+.counter {
+  display: flex;
+  list-style-type: none;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+}
+
+.canopy-text {
+  display: flex;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.canopy {
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.info-section {
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+}
+
+.about-info {
+  max-width: 37.5rem;
+  margin: 1.5rem auto;
+}
+
+.info-container {
+  max-width: 37.5rem;
+  margin: 1.5rem auto;
+}
+
+.primary-text {
+  font-family: poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.btn {
+  color: #396df2;
+  background: #efebeb;
+}
+
+.see-project-button {
+  color: #396df2;
+  font-family: poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border-radius: 25px;
+  border: 2px solid #396df2;
+  padding: 5px;
+  width: 200px;
+  height: 150px;
+}
+
+.list-frames {
+  list-style-type: none;
+  left: 12px;
+  top: 12px;
+  border-radius: 8px;
+}
+
+.language-framework {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+}
+
```

Comment:
- [ ] In this button there are styles missing, please add the ones needed to make this looks similar to the one on the FIGMA template 💯 

![image](https://user-images.githubusercontent.com/93630700/205455167-a5ff88d3-bc08-495f-99d3-3bcf47fadac0.png)

![image](https://user-images.githubusercontent.com/93630700/205455175-a45f1dec-588c-4f92-bab7-43c8ce414351.png)